src/capitalize.spec.js

```diff
@@ -0,0 +1,21 @@
+import capitalize from "./capitalize"
+
+
+describe('Test suite for "capitalize" funtion', () => {
+    it('Should convert "hello" to "Hello"', () => { expect(capitalize('hello')).toBe('Hello') })
+    it('Should apply only to first word if passed several words', () => {
+        expect(capitalize('several words written')).toBe('Several words written')
+    })
+    describe('Testing string with non-letter characters', () => {
```

Comment:
It was not obvious why you are testing this. I guess you want to ensure that special character at the start of a string stay unchanged. I would rephrase the name or add a comment I think.
```suggestion
    describe('with strings starting with special characters', () => {
```